lib/puppet/type/concat_file.rb

```diff
@@ -58,68 +54,98 @@ def exists?
   end
 
   newparam(:order) do
-    desc "Controls the ordering of fragments. Can be set to alphabetical or numeric."
-    defaultto 'numeric'
+    desc "Controls the ordering of fragments. Can be set to :alpha or :numeric."
```

Comment:
It does raise the question what the actual purpose of the target parameter is then, because it's not used for anything except emitting that warning.